Pipfile

```diff
@@ -0,0 +1,44 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[dev-packages]
+django-debug-toolbar = "*"
+django-debug-toolbar-request-history = "*"
+django-naomi = "*"
+fixmydjango = "*"
+model-mommy = "*"
+pre_commit = "*"
+
+
+
+[packages]
+
+Django = "==1.11.8"
+celery = {extras = ["redis"]}
+django-model-utils = "*"
+django-webpack-loader = "*"
+django-js-reverse = "*"
+django-import-export = "*"
+python-decouple = "<=4.0"
```

Comment:
I've always wondered, why are we doing this? Either we do that for all libs or none at all.

Reply:
Better to open another issue to discuss that since the problem is not related to pipenv

Reply:
discussion about that here #164